pkg/operator/ceph/csi/spec.go

```diff
@@ -521,11 +526,12 @@ func deleteCSIDriverResources(
 	return succeeded
 }
 
-func applyCephClusterNetworkConfig(objectMeta *metav1.ObjectMeta, rookclientset rookclient.Interface) (bool, error) {
+func applyCephClusterNetworkConfig(ctx *clusterd.Context, objectMeta *metav1.ObjectMeta) (bool, error) {
```

Comment:
`context` is more common in the rook codebase for this var
```suggestion
func applyCephClusterNetworkConfig(context *clusterd.Context, objectMeta *metav1.ObjectMeta) (bool, error) {
```

Reply:
in this function we use `context.TODO()` from package `context` so used `ctx` here